src/DesktopAnalytics/Analytics.cs

```diff
@@ -230,14 +230,31 @@ public static void Track(string eventName, Dictionary<string, string> properties
 		/// <param name="e"></param>
 		public static void ReportException(Exception e)
 		{
+			ReportException(e, null);
```

Comment:
I thought about using an optional parameter, but doing so breaks ABI compatibility, which may not be an important consideration in this circumstance.  (and I mean the B for Binary in ABI, although i'm not sure that's exactly the right term for the concept I was reading about in the discussion of optional parameters.)